mmcv/image/transforms/geometry.py

```diff
@@ -22,6 +22,22 @@ def imflip(img, direction='horizontal'):
         return np.flip(img, axis=0)
 
 
+def imflip_(img, direction='horizontal'):
+    """Flip an image horizontally or vertically by cv2 inplace flip.
+    Args:
+        img (ndarray): Image to be flipped.
+        direction (str): The flip direction, either "horizontal" or "vertical".
+
+    Returns:
+        ndarray: The flipped image(inplace).
+    """
+    assert direction in ['horizontal', 'vertical']
+    if direction == 'horizontal':
+        return cv2.flip(img, 1, img)
+    else:
+        return cv2.flip(img, 0, img)
```

Comment:
How about the speed of `cv2.flip(img, 1)` and `np.flip(img, axis=1)`?

Reply:
np.flip is a lazy operation. But it will pay eventually because of it makes array non-contiguous.

cv2 inplace flip is fast, and do not have contiguous issue.

Benchmark (compare with np.flip().copy()) see https://github.com/innerlee/cvbenchmark/blob/master/benchmarks/run_flip.py
and the results:

```bash
./mmbench benchmarks/run_flip.py
# flip cv2  500 loops, best of 5: 915 usec per loop
# flip np   50 loops, best of 5: 7.66 msec per loop
```